ppr-ui/src/views/newMhrRegistration/YourHome.vue

```diff
@@ -25,17 +43,15 @@
 
     <section id="mhr-home-certification" class="mt-10">
       <h2>Home Certification</h2>
-      <p class="mt-2">
-        Enter either the Canadian Standards Association (CSA) number OR the Engineer's inspection information.
-      </p>
+      <p class="mt-2" data-test-id="home-certification-prompt"> {{ homeCertificationPrompt }} </p>
 
       <HomeCertification
         :validate="validateCertification"
         :class="{'border-error-left': validateCertification}"
       />
     </section>
 
-    <section id="mhr-rebuilt-status" class="mt-10">
+    <section id="mhr-rebuilt-status" class="mt-10" v-if="showRebuiltStatus">
```

Comment:
usually we lead with the conditional before other props/classes/events etc.
ie `<section v-if="" ....>`

Super minor but just for consistency.